views/github-create-branch.hbs

```diff
@@ -21,15 +21,56 @@
 <input type="hidden" id="_csrf" name="_csrf" value="{{csrfToken}}" />
 <input type="hidden" id="jiraHost" name="jiraHost" value="{{jiraHost}}" />
 
-{{> navigation hideBackButton=true }}
-
 <section class="gitHubCreateBranch">
   <div class="headerImage">
     <img src="/public/assets/jira-and-github.png" alt="Jira and GitHub logos" />
   </div>
   <div class="gitHubCreateBranch__title">
-    Create a Branch
+    <div class="gitHubCreateBranch__header">Create GitHub Branch</div>
+    <div class="gitHubCreateBranch__subHeader">
+      Creating a branch for
+      <!--TODO: Need to add icons based on the type of issue(story/task/bug) -->
+      <a href="#">KEY-1</a>
+    </div>
   </div>
+
+  <form id="createBranchForm" class="aui top-label">
+    <!--TODO: Add condition to display this only for GHES -->
+    <div class="field-group">
+      <aui-label for="server">GitHub Enterprise Server instance</aui-label>
+      <select id="ghServers" name="server" class="full-width-field" placeholder="Select a repository">
+        {{#each servers}}
+          <option value="{{id}}">{{server}} / {{appName}}</option>
+        {{/each}}
+      </select>
+    </div>
+    <div class="field-group">
+      <aui-label for="repo">Repository</aui-label>
+      <select id="ghRepo" name="repo" class="full-width-field" placeholder="Select a repository">
+        {{#each repos}}
+          <option value="{{id}}">{{org}} / {{name}}</option>
+        {{/each}}
+      </select>
+    </div>
+    <div class="field-group">
+      <aui-label for="branch">Repository</aui-label>
+      <select id="ghParentBranch" name="parentBranch" class="full-width-field" placeholder="Select a branch">
+        {{#each branches}}
+          <option value="{{id}}">{{name}}</option>
+        {{/each}}
+      </select>
+    </div>
+    <div class="field-group">
+      <label for="branch-name">Branch name</label>
```

Comment:
Using `aui-label` for the other fields and the font size looks a little bit smaller here.